### DISCUSSION TITLE
How does active-timestamp work ?

### DISCUSSION THREAD

**Discussion by apraga** [alphapapa/org-ql#443] [Q&A] [answered]
hi,

I very much like your package but am confused about active timestamp. My intuition was that it would recognize timestamp like `<...>` and not `[ ...]`.

But with a single file in org-agend-file like this 

```
#+title: Inbox
#+filetags: inbox
* TODO Écrire formule 
SCHEDULED: <2024-07-04 jeu.>
* TODO test
DEADLINE: <2024-07-02 mar.> SCHEDULED: <2024-07-02 mar.>

```

Querying (ts-active :on today) returns nothing. Using (ts :on today) returns the first item (todays the 4th of July). Am I missing something ? Thanks

**Comment by alphapapa** [comment, marked as answer]
It works for me:

![works](https://github.com/alphapapa/org-ql/assets/601365/f54bed84-003f-4f5f-be97-116b5af93308)

You probably need to upgrade.  See https://github.com/alphapapa/org-ql/discussions/429

**Comment by apraga** [comment]
Thanks, I was wondering it was due to the French notation for date but could not find this issue. It works with an update :)
